### PR TITLE
Prepare for v4.5.0

### PR DESCRIPTION
**Pull request by nchen63**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,56 @@
...
+- [new-rule-option] `no-inferrable-types` adds option  `ignore-properties` (#2178)
+- [new-rule-option] `typedef` adds options `object-destructuring` and `array-destructuring` options (#2146)
+- [new-rule-option] `member-ordering` adds option `alphabetize` (#2101)
+- [new-rule-option] `no-trailing-whitespace` adds option `ignore-jsdoc` (#2177)
+- [new-rule-option] `no-trailing-whitespace` adds option `ignore-comments option` (#2153)
+- [new-fixer] `no-inferrable-types` automatically remove inferrable type annotations (#2178)
+- [new-fixer] `no-any` (#2165)
+- [new-fixer] `noConsecutiveBlankLines` (#2201)
+- [new-fixer] `object-literal-shorthand` (#2165)
+- [bugfix] `no-switch-case-fallthrough` Don't exit when a rule requires type checking but type checking is not enabled (#2188)
```
Comment (ajafff) [inline CHANGELOG.md]: That's not specific to that rule and should be highlighted as a notable change.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,56 @@
...
+- [new-rule] `no-non-null-assertion` (#2221)
+- [new-rule] `ban-types` (#2175)
+- [new-rule] `no-duplicate-super` (#2038)
+- [new-rule] `newline-before-return` (#2173)
+- [new-rule-option] `completed-docs` adds options for location, type, and privacy. Also adds interfaces, enums, types (#2095)
+- [new-rule-option] `no-inferrable-types` adds option  `ignore-properties` (#2178)
+- [new-rule-option] `typedef` adds options `object-destructuring` and `array-destructuring` options (#2146)
+- [new-rule-option] `member-ordering` adds option `alphabetize` (#2101)
+- [new-rule-option] `no-trailing-whitespace` adds option `ignore-jsdoc` (#2177)
+- [new-rule-option] `no-trailing-whitespace` adds option `ignore-comments option` (#2153)
```
Comment (ajafff) [inline CHANGELOG.md]: Misplaced closing backtick

**Comment by adidahiya** [comment]
@nchen63 no update of `tslint:latest`?

**Comment by nchen63** [comment]
forgot :(

**Comment by adidahiya** [comment]
@nchen63 yep, let's do 4.5.1. I filed #2263

**Pull request merged**